Extensions/AdvancedP2PEventHandling-header.json

```diff
@@ -0,0 +1,15 @@
+{
+  "shortDescription": "Adds a way to handle events the advanced way. It is more complex but also potentially more performant.",
```

Comment:
"events" can be confusing for a new user, let's be more specific:

`Allows handling all received P2P events at once instead of one per frame`